e2e-tests/src/test/azero_to_eth.rs

```diff
@@ -1,93 +1,197 @@
-use aleph_client::contract::ExecCallParams;
-use anyhow::{Error, Result};
+use anyhow::Result;
 use ethers::utils;
 use log::info;
 
-use crate::{
-    azero,
-    config::{setup_test, TestContext},
-    wait::wait_for_balance_change,
-};
+use crate::{client::Client, config::setup_test, wait::wait_for_balance_change};
 
 /// One-way `Aleph Zero` -> `Ethereum` transfer through `most`.
 /// Requires a prior transaction in the other direction to have completed.
 /// This is easily done by running the test for the other direction first.
 /// Approves the `most` contract to use the wETH funds.
 /// Burns the required funds in the wETH contract on Aleph Zero.
-/// Transfers `transfer_amount` of burned wETH over the bridge, unwrapping the transfer to a specified Ethereum account.
+/// Transfers `transfer_amount` of burned wETH over the bridge.
 /// Waits for the transfer to complete.
-/// Verifies that the correct amount of ETH is present on the Ethereum chain.
 /// It relies on all the relevant contracts being deployed on both ends and the (wETH_ETH:wETH_AZERO) pair having been added to `most`.
```

Comment:
Suggestion: use numbered list here for test steps.

Reply:
removed the steps altogether, as @obrok suggested